compliance_suite/cli.py

```diff
@@ -51,15 +77,11 @@ def report(server: str,
         uptime (int): The local server duration in seconds. Default - 3600 seconds
     """
 
-    if server is None:
-        raise Exception("No server provided. Please provide a server URL.")
-
-    if version is None:
-        raise Exception("No version provided. Please provide a version.")
```

Comment:
Using `click`'s `prompt` ensures that a value is provided in the option. So, we can remove these checks.